cmd/ocm/create/cluster/cmd.go

```diff
@@ -377,6 +394,68 @@ func getVersionOptionsWithDefault(connection *sdk.Connection, channelGroup strin
 	return
 }
 
+func getSubscriptionTypeOptions(connection *sdk.Connection) ([]arguments.Option, error) {
+	options := []arguments.Option{}
+	billingModels, err := getBillingModels(connection)
+	if err != nil {
+		return options, err
+	}
+	for _, billingModel := range billingModels {
+		//Standard billing model should always be the first option
+		if billingModel.Id() == StandardSubscriptionType {
+			standardOption := arguments.Option{
+				Value: fmt.Sprintf("%s (%s)", billingModel.Id(), billingModel.Description()),
+			}
+			if len(options) == 0 { // nil or empty slice or after last element
+				options = append(options, standardOption)
+			} else {
+				options = append(options[:1], options[0:]...)
+				options[0] = standardOption
+			}
+		} else {
+			options = append(options, arguments.Option{
+				Value: fmt.Sprintf("%s (%s)", billingModel.Id(), billingModel.Description()),
+			})
+		}
+	}
+	return options, nil
+}
+
+func getSubscriptionTypeIdFromDescription(connection *sdk.Connection, value string) string {
```

Comment:
I would suggest using the same approach used for subnets, see https://github.com/openshift-online/ocm-cli/blob/main/cmd/ocm/create/cluster/cmd.go#L885

Reply:
Used similar methods as subnets